src/leader_scheduler.rs

```diff
@@ -56,9 +56,9 @@ pub struct LeaderScheduler {
     // This value must be divisible by ticks_per_slot
     pub ticks_per_epoch: u64,
 
-    // The length of time in ticks for which a vote qualifies a candidate for leader
+    // The number of slots for which a vote qualifies a candidate for leader
     // selection
-    active_window_tick_length: u64,
+    active_window_length: u64,
```

Comment:
nit: please keep the units: `active_window_slot_length`.  It's error prone otherwise, and the self-documentation is nice

Reply:
should I call it active_window_num_slots, slot length sounds like the number of ticks in a slot

Reply:
Either way, can you do it in a follow-up PR? I want this one merged.

Reply:
I'm doing it. I want to build off of this and I don't mind the churn of another name change.